dictionary.rb

```diff
@@ -0,0 +1,100 @@
+require_relative 'dictionary_loader'
+require_relative 'dictionary_analyzer'
+
+class Dictionary
+  attr_reader :dictionary, :analyzer, :search_results
+  def initialize
+    print "What is the name of the dictionary you want to load?\n > "
+    dictionary_name = gets.chomp
+    check_for_quit dictionary_name
```

Comment:
Someone likes to be user friendly with check_for_quit. I never let my users escape.